rmc-docs/src/dashboard.md

```diff
@@ -0,0 +1,11 @@
+# RMC Dashboard
+
+The [RMC Dashboard](./dashboard/index.html) purpose is to show the level of support in RMC for all Rust features.
+To this end, we extract all examples from the following books:
```

Comment:
Maybe more description, like "all Rust code snippet examples from the following general Rust documentation books"

Reply:
I extended it a bit but would like to extend and review this in a follow-up PR, so opened #509